index.js

```diff
@@ -967,7 +955,7 @@ function buildArray (location, code, name, key = null) {
   if (schema.nullable) {
     code += `
       if(obj === null) {
-        return '${$asNull()}';
+        return 'null';
```

Comment:
Why this change?

Reply:
Same here - the $asNull function is now inside a string constant, so it is not available to be called in this context.